doc/release-notes/0.25/0.25.md

```diff
@@ -0,0 +1,110 @@
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](https://eclipse.org/openj9/docs/openj9_support/index.html).
+
+
+## Notable changes in this release
+
+The following table covers notable changes in v0.25.0. Further information about these changes can be found in the [user documentation](https://www.eclipse.org/openj9/docs/version0.25/).
+
+<table cellpadding="4" cellspacing="0" summary="" width="100%" rules="all" frame="border" border="1"><thead align="left">
+<tr valign="bottom">
+<th valign="bottom">Issue number</th>
+<th valign="bottom">Description</th>
+<th valign="bottom">Version / Platform</th>
+<th valign="bottom">Impact</th>
+</tr>
+</thead>
+<tbody>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/issues/10620">#10620</a></td>
+<td valign="top">New option to support JEP 390: Warnings for value-based classes</td>
+<td valign="top">OpenJDK 16</td>
+<td valign="top">OpenJ9 adds option <strong>-XX:DiagnoseSyncOnValueBasedClasses=&lt;number&gt;</strong> for compatibility with the reference implementation. A value of <tt>1</tt> generates a <tt>VirtualMachineError</tt> error and a value of <tt>2</tt> prints a warning message.</td>
+</tr>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/pull/8847">#8847</a></td>
+<td valign="top">Single build for compressed references and non-compressed references</td>
+<td valign="top">All versions</td>
+<td valign="top">A single build now supports both compressed references and non-compressed references. The object reference mode is selected at run time based on the specified heap size (<strong>-Xmx</strong>) or by using command-line options that control the selection of compressed references. If compressed references mode is automatically selected and you don't want to use compressed references, turn it off using the command-line option <strong>-Xnocompressedrefs</strong>.</td>
+</tr>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/pull/11624">#11624</a></td>
+<td valign="top">Enabling zlibNX hardware-accelerated data compression and decompression on AIX&reg;</td>
+<td valign="top">All versions</td>
+<td valign="top">On AIX systems that contain the Nest accelerator (NX) co-processor, OpenJ9 now uses the <tt>zlibNX</tt> library instead, if it is installed. The NX co-processor is available on IBM POWER9&reg; systems.</td>
+</tr>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/omr/pull/5770">#5770</a></td>
+<td valign="top">z/OS&reg; support for the <strong>%sysname</strong> dump token</td>
+<td valign="top">All versions</td>
+<td valign="top">The <strong>%sysname</strong> dump token is added on z/OS, which equates to the <strong>SYSNAME sysparm</strong>.</td>
+</tr>
+</table>
+
+
+## Known Issues
+
+The v0.24.0 release contains the following known issues and limitations:
```

Comment:
I guess this should be `0.25.0`?

Reply:
Thanks for spotting @PascalSchumacher - I've pushed a fix.